### ISSUE TITLE
Missing PDF referenced in `website/course_bootstrap/python.ipynb`

### ISSUE THREAD

**Issue by douglatornell** [rhwhite/numeric_2022#10]
```
website/getting_started/python.ipynb:208:
 WARNING: File not found: 'getting_started/pdfs/Python-data-manipulations.pdf'
```

**Comment by SusanEAllen** [comment]
Where so I find python.ipynb to see the name of the pdf file that is missing?

**Comment by douglatornell** [comment]
PDF name is in the issue description above:
```website/getting_started/python.ipynb:208:
 WARNING: File not found: 'getting_started/pdfs/Python-data-manipulations.pdf'
```
I'll push the notebook as soon as I've got other issues sorted out.
Or you can look at it in `numeric_2022/course_bootstrap/python.ipynb` on the `gh-pages` branch.

**Comment by SusanEAllen** [comment]
drop that line.

**Comment by douglatornell** [comment]
resolved by 8c1a6a4
